cobalt/browser/metrics/cobalt_metrics_services_manager.h

```diff
@@ -33,10 +33,16 @@ class CobaltMetricsServicesManager {
  public:
   // Static Singleton getter for metrics services manager.
   static metrics_services_manager::MetricsServicesManager* GetInstance() {
-    static auto instance = new metrics_services_manager::MetricsServicesManager(
+    const auto instance = new metrics_services_manager::MetricsServicesManager(
```

Comment:
Wait - this would return a newly created instance now every time when it's called, no ?

Reply:
Let's talk about the static vs non-static approach offline. For now, this PR should be fine to go in as-is. If we find a path forward without static, happy to refactor it.